Compilation error: NC1002: Unknown method: object.Object()

I am using the N3 plugin _ngd-seattle.neo-blockchain-toolkit_ v2.0.5 with C# and Visual Studio Code on Windows 11.  I am getting this error from _nccs_ v3.0.0.0-rc3:

NC1002: Unknown method: object.Object()

There is no line information or other details than that so I have no idea what the problem is.  This occurs using the command:

error MSB3073: The command "nccs --debug --no-optimize

--- a/src/Neo.Compiler.CSharp/MethodConvert.cs
+++ b/src/Neo.Compiler.CSharp/MethodConvert.cs
@@ -312,13 +312,13 @@ namespace Neo.Compiler
 
         private void ProcessConstructorInitializer(SemanticModel model)
         {
+            INamedTypeSymbol type = Symbol.ContainingType;
+            if (type.IsValueType) return;
+            INamedTypeSymbol baseType = type.BaseType!;
+            if (baseType.SpecialType == SpecialType.System_Object) return;
             ConstructorInitializerSyntax? initializer = ((ConstructorDeclarationSyntax?)SyntaxNode)?.Initializer;
             if (initializer is null)
             {
-                INamedTypeSymbol type = Symbol.ContainingType;
-                if (type.IsValueType) return;
-                INamedTypeSymbol baseType = type.BaseType!;
-                if (baseType.SpecialType == SpecialType.System_Object) return;
                 IMethodSymbol baseConstructor = baseType.InstanceConstructors.First(p => p.Parameters.Length == 0);
                 if (baseType.DeclaringSyntaxReferences.IsEmpty && baseConstructor.GetAttributes().All(p => p.AttributeClass!.ContainingAssembly.Name != "Neo.SmartContract.Framework"))
                     return;

--- a/tests/Neo.Compiler.CSharp.UnitTests/TestClasses/Contract_Polymorphism.cs
+++ b/tests/Neo.Compiler.CSharp.UnitTests/TestClasses/Contract_Polymorphism.cs
@@ -5,6 +5,8 @@ namespace Neo.Compiler.CSharp.UnitTests.TestClasses
 {
     public abstract class A : SmartContract.Framework.SmartContract
     {
+        public A() : base() { }
+
         public int sum(int a, int b)
         {
             return a + b;
@@ -16,6 +18,8 @@ namespace Neo.Compiler.CSharp.UnitTests.TestClasses
 
     public abstract class B : A
     {
+        public B() : base() { }
+
         public int mul(int a, int b)
         {
             return a * b;

